File: Serene/Serene.Script/Imports/ServerImports/Northwind.OrderShippingState.cs
using jQueryApi;
using Serenity;
using Serenity.ComponentModel;
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Serene.Northwind
{
    [EnumKey("Northwind.OrderShippingState"), PreserveMemberCase]
    public enum OrderShippingState
    {
        NotShipped = 0,
        Shipped = 1
    }
}

File: Serene/Serene.Core/Imports/ServerTypings/Northwind.OrderShippingState.ts
namespace Serene.Northwind {
    export enum OrderShippingState {
        NotShipped = 0,
        Shipped = 1
    }
    Serenity.Decorators.registerEnumType(OrderShippingState, 'Serene.Northwind.OrderShippingState', 'Northwind.OrderShippingState');
}
